pkg/plugins/plugins.go

```diff
@@ -199,17 +200,36 @@ func GetAll(imp string) ([]*Plugin, error) {
 		err error
 	}
 
-	chPl := make(chan plLoad, len(pluginNames))
+	driverExist := func(dn string) bool {
+		_, ok := storage.plugins[dn]
+		return ok
+	}
+
+	chPl := make(chan *plLoad, len(pluginNames))
+	var wg sync.WaitGroup
 	for _, name := range pluginNames {
+		if driverExist(name) {
+			chPl <- nil
```

Comment:
The plugins that have been loaded already are in the storage.plugins array, so they can be skipped. Was I interpeting that question correctly?